### ISSUE TITLE
No longer getting Windows Notifications

### ISSUE THREAD

**Issue by Seany1990** [runelite/runelite#18466]
### Type

Incorrect behavior

### Operating System

Windows

### OS Version

10

### Bug description

I have windows notifications enabled in settings
I have made no windows system changes since I have last known the issue to work correctly
I have made no changes to runelite settings
I am getting the same problem on both of my computers

### RuneLite version

runelite: 1.10.41.1
launcher: 2.6.10
windows 10 pro 10.0.19045 - 19045

**Comment by cdfisher** [comment]
Can you show a picture of your notification settings?

**Comment by Seany1990** [comment]
![rlscreen](https://github.com/user-attachments/assets/8c5b6988-1939-4cd0-a9a2-59d5818fed35)

Since making the ticket I have made a new profile with no extra plugin hub settings, I only changed the notif settings to what is shown in the screenshot. I again did not get a notification.

I do get the windows ping and the screen flashes, its the message in the notification pane that doesnt show. This is true for both scenarios, with the fresh test profile and my own profile
